evalml/pipelines/components/utils.py

```diff
@@ -163,6 +163,45 @@ def handle_component_class(component_class):
     return component_class
 
 
+def drop_natural_language_columns(X):
```

Comment:
Thanks for doing this! It'd be great if you could add test cases for this function as well as `set_boolean_columns_to_categorical`.